packages/commonwealth/client/scripts/state/api/communityStake/editCommunityStake.ts

```diff
@@ -0,0 +1,33 @@
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+import app from 'state';
+
+interface EditCommunityStakeProps {
+  communityId: string;
+  stakeId: number;
+}
+
+const editCommunityStake = async ({
+  communityId,
+  stakeId,
+}: EditCommunityStakeProps) => {
+  return await axios.put(
+    `${app.serverUrl()}/communityStakes/${communityId}/${stakeId}`,
+    {
+      jwt: app.user.jwt,
+      community_id: communityId,
+      stake_id: stakeId,
+      stake_token: '',
+      stake_scaler: 1,
+      stake_enabled: true,
```

Comment:
is it required to define this?

Reply:
It was required previously due to a bug in my backend. I had fixed it here:
https://github.com/hicommonwealth/commonwealth/pull/6468